docs/src/development/email.md

```diff
@@ -45,6 +45,13 @@ When SMTP support is enabled the environment variable `PLATFORM_SMTP_HOST` will
 Changing the SMTP status will not take effect immediately.  You will need to issue a new *build*, not just a new deploy, for the changes to take effect. That means making a trivial change to some file in Git for the application.
 {{< /note >}}
 
+## Ports
```

Comment:
Why reorder?

Reply:
Structure-wise, I thought it made more sense to have all general information before the language specific ones rather than [general stuff] [PHP specific] [Java specific] [general stuff].

Reply:
ACK